### PR TITLE
[Docs] Add opset6 content and add opset reference

### PR DESCRIPTION
**Pull request by andrei-kochin**
### Details:
Delta to opset5:
 - CTCGreedyDecoderSeqLen
 - GatherElements
 - MVN-1 -> MVN-6
 - ExperimentalDetectron*
 - ReadValue-3 -> ReadValue-6
 - Assign-3 -> Assign-6

### Tickets:
 - #42541

### CODE REVIEW

File: docs/ops/opset.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ This topic provides a complete list of available sets of operations supported in
 
 | OpenVINO™ Version | Actual Operations Set            | 
 | :---------------- | :------------------------------- | 
+| 2021.3            | [opset6](opset6.md)   |
```
Comment (lazarevevgeny) [inline docs/ops/opset.md:9]: Please, update the `./docs/doxygen/ie_docs.xml` line 85 as well.

File: docs/ops/opset6.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+* [Cos](arithmetic/Cos_1.md)
+* [Cosh](arithmetic/Cosh_1.md)
+* [CumSum](arithmetic/CumSum_3.md)
+* [DeformableConvolution](convolution/DeformableConvolution_1.md)
+* [DeformablePSROIPooling](detection/DeformablePSROIPooling_1.md)
+* [DepthToSpace](movement/DepthToSpace_1.md)
+* [DetectionOutput](detection/DetectionOutput_1.md)
+* [Divide](arithmetic/Divide_1.md)
+* [Elu](activation/Elu_1.md)
+* [EmbeddingBagOffsetsSum](sparse/EmbeddingBagOffsetsSum_3.md)
```
Comment (lazarevevgeny) [inline docs/ops/opset6.md:47]: Please, add ExperimentalDetectron operations

File: docs/ops/opset6.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+
+## Table of Contents <a name="toc"></a>
+
+* [Abs](arithmetic/Abs_1.md)
+* [Acos](arithmetic/Acos_1.md)
+* [Acosh](arithmetic/Acosh_3.md)
+* [Add](arithmetic/Add_1.md)
+* [Asin](arithmetic/Asin_1.md)
+* [Asinh](arithmetic/Asinh_3.md)
+* [Assign](infrastructure/Assign_3.md)
```
Comment (lazarevevgeny) [inline docs/ops/opset6.md:18]: Assign-6

File: docs/ops/opset6.md [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+* [Parameter](infrastructure/Parameter_1.md)
+* [Power](arithmetic/Power_1.md)
+* [PReLU](activation/PReLU_1.md)
+* [PriorBoxClustered](detection/PriorBoxClustered_1.md)
+* [PriorBox](detection/PriorBox_1.md)
+* [Proposal](detection/Proposal_4.md)
+* [PSROIPooling](detection/PSROIPooling_1.md)
+* [Range](generation/Range_4.md)
+* [ReLU](activation/ReLU_1.md)
+* [ReadValue](infrastructure/ReadValue_3.md)
```
Comment (lazarevevgeny) [inline docs/ops/opset6.md:109]: ReadValue-6

**Review by lazarevevgeny — CHANGES_REQUESTED** [review]

File: docs/ops/opset6.md (hunk 3 above)
Comment (andrei-kochin) [reply docs/ops/opset6.md:18]: I can't find the 6th version of the specification

~/repo/openvino/docs/ops$ find . | grep -i assign
./infrastructure/Assign_3.md

File: docs/ops/opset6.md (hunk 4 above)
Comment (andrei-kochin) [reply docs/ops/opset6.md:109]: I can't find the 6th version of the specification
~/repo/openvino/docs/ops$ find . | grep -i readva
./infrastructure/ReadValue_3.md

File: docs/ops/opset6.md (hunk 2 above)
Comment (andrei-kochin) [reply docs/ops/opset6.md:47]: No specs are yet present so to be added once they are available

File: docs/ops/opset.md (hunk 1 above)
Comment (andrei-kochin) [reply docs/ops/opset.md:9]: Done

**Review by lazarevevgeny — APPROVED** [review]

**Pull request merged**